packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -0,0 +1,63 @@
+import { LightningElement, api } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+
+
+export default class QuanticTab extends LightningElement {
+  /** @type {import("coveo").Tab} */
+  tab;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isActive = false;
```

Comment:
I think the value by default is false
What do you think?

Reply:
By default it would be undefined I think but yes at the end both are falsy values so i could avoid setting it to false.